disperser/dataapi/subgraph_client.go

```diff
@@ -182,9 +182,10 @@ func (sc *subgraphClient) QueryNumBatchesByOperatorsInThePastBlockTimestamp(ctx
 	return numBatchesByOperator, nil
 }
 
-func (sc *subgraphClient) QueryIndexedDeregisteredOperatorsInTheLast14Days(ctx context.Context) (*IndexedDeregisteredOperatorState, error) {
-	last14Days := uint64(time.Now().Add(-_14Days).Unix())
-	deregisteredOperators, err := sc.api.QueryDeregisteredOperatorsGreaterThanBlockTimestamp(ctx, last14Days)
+func (sc *subgraphClient) QueryIndexedDeregisteredOperatorsForTimeWindow(ctx context.Context, days int32) (*IndexedDeregisteredOperatorState, error) {
+	// Query all deregistered operators in the last N days.
+	lastNDays := uint64(time.Now().Add(-time.Duration(days) * 24 * time.Hour).Unix())
```

Comment:
`lastNDaysInSecs`, the current name would suggest the time unit is days (which it is not)